packages/flame_forge2d/lib/forge2d_game.dart

```diff
@@ -23,6 +23,20 @@ class Forge2DGame extends FlameGame {
 
   @override
   void update(double dt) {
+    switch (gameLifecycleState) {
```

Comment:
I think this could move to a separate private method in `FlameGame` and then have a flag `pauseWhenBackgrounded` or something (open to other name suggestions, not a big fan of that name). And that flag should default to false, so that this won't be a breaking change.
In this private method should use `pauseEngine` and `resumeEngine` instead of using an early `return`.
Maybe `foregroundedFirstUpdate` won't be a problem if those two methods are used instead either?